t/31-client_job_groups.t

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>..
+
+use Mojo::Base -strict;
+
+use FindBin;
+use lib "$FindBin::Bin/lib", "$FindBin::Bin/../external/os-autoinst-common/lib";
+
+use Test::More;
+use Mojo::Server::Daemon;
+use Mojo::JSON qw(decode_json);
+use OpenQA::Script::Client;
+use OpenQA::Test::Case;
+use OpenQA::Test::TimeLimit '10';
+
+OpenQA::Test::Case->new->init_data(fixtures_glob => '03-users.pl');
+
+# Mock WebAPI with extra test routes
+my $daemon = Mojo::Server::Daemon->new(listen => ['http://127.0.0.1']);
+my $app    = $daemon->build_app('OpenQA::WebAPI');
+$app->log->level('error');
+my $port = $daemon->start->ports->[0];
+my $host = "http://127.0.0.1:$port";
+
+my %options = (host => $host);
+
+my $template_openqa = "$FindBin::Bin/data/job-templates/openqa.yaml";
+my @common_args     = qw(job_templates_scheduling/1 post schema=JobTemplates-01.yaml preview=1);
+my $ret             = '';
+is $ret, '', 'Initial test';
+$ret = run(\%options, qw(job_groups post name=Test));
+warn "RET is [$ret]";
```

Comment:
this looks like temporary debugging code?

Reply:
Yes. And it's still not working. I'm dropping that file and look for a different solution to test this.